docs/handshake.md

```diff
@@ -0,0 +1,1525 @@
+
+**NOTE:** The `TsbPd Resv` and `TsbPdDelay` fields both refer to latency,
+but the use is different in HSv4 and HSv5.
+
+In HSv4, only the lower 16 bits (`TsbPdDelay`) are used. The upper 16 bits
+(`TsbPd Resv`) are simply unused. There's only one direction, so `HSREQ` is
+sent by the Sender, `HSRSP` by the Receiver.  `HSREQ` contains only the Sender
+latency, and `HSRSP` contains only the Receiver latency.
+
+This is different from HSv5, in which the latency value for the sending
+direction in the lower 16 bits (`SndTsbPdDelay`, 16 - 31 in network order) and
+for receiving direction is placed in the upper 16 bits (`RcvTsbpdDelay`, 0 -
+15). The communication is bidirectional, so there are two latency values, one
+per direction. Therefore both HSREQ and HSREQ messages contain both the Sender
+and Receiver latency values.
+
+[Return to top of page](#srt-handshake)
+
+
+### SRT Extension Commands
+
+#### HSREQ and HSRSP
+
+The `SRT_CMD_HSREQ` message contains three 32-bit fields designated as:
+
+- `SRT_HS_VERSION`: string (0x00XXYYZZ) representing SRT version XX.YY.ZZ
+- `SRT_HS_FLAGS`: the SRT flags (see below)
+- `SRT_HS_LATENCY`: the latency specification
+
+```
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                    SRT Version {>=10300h}                     |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                           SRT Flags                           |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |(HSv4)  TsbPd Resv = 0         |     TsbPdDelay {20..8000}     |
+   |(HSv5) RcvTsbPdDelay {20..8000}|   SndTsbPdDelay {20..8000}    |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+The flags (`SRT Flags` field) are the following bits, in order:
+
+(0) `SRT_OPT_TSBPDSND`: The party will be sending in TSBPD (Time Stamp Based 
+Packet Delivery) mode.
+
+This is used by the Sender party to specify that it will use TSBPD mode.
+The Responder should respond with its setting for TSBPD reception; if it isn't 
+using TSBPD for reception, it responds with its reception TSBPD flag not set. 
+In HSv4, this is only used by the Initiator.
+
+(1) `SRT_OPT_TSBPDRCV`: The party expects to receive in TSBPD mode.
+
+This is used by a party to specify that it expects to receive in TSBPD mode.
+The Responder should respond to this setting with TSBPD sending
+mode (HSv5 only) and set the sending TSBPD flag appropriately. In HSv4 this is 
+only used by the Responder party.
+
+(2) `SRT_OPT_HAICRYPT`: The party includes `haicrypt` (legacy flag).
+
+This **special legacy compatibility flag** should be always set. See below 
+for more details.
+
+(3) `SRT_OPT_TLPKTDROP`: The party will do TLPKTDROP.
+
+Declares the `SRTO_TLPKTDROP` flag of the party. This is important
+because both parties must cooperate in this process. In HSv5, if both
+directions are TSBPD, both use this setting. This flag must always be
+ set in live mode.
```

Comment:
It's not true that this flag must be always set in live mode, although this is default and most recommended.

Reply:
Change last sentence (lines 1181-1182) to:
"While it is not always necessary to set this flag to live mode, it is the default and most recommended setting."